cmd/containerd-shim-runc-v2/runc/platform.go

```diff
@@ -139,6 +139,10 @@ func (p *linuxPlatform) CopyConsole(ctx context.Context, console console.Console
 		go func() {
 			cwg.Done()
 			io.Copy(outW, epollConsole)
+			// if stdin is not nil epollConsole will be closed
```

Comment:
Does this comment match what we're checking? It seems like this is more "if stdin is empty ..."

Reply:
done.thank you.